### DISCUSSION TITLE
I saw a very big change with the Lighthouse tests, in January 2024, you definitely changed and increased the criteria in Lighthouse,

### DISCUSSION THREAD

**Discussion by earthtopomaps** [GoogleChrome/lighthouse#15755] [General] [answered]
I saw a very big change with the Lighthouse tests, in January 2024, you definitely changed and increased the criteria in Lighthouse, in December 2023 I had passed 36 audits, now I only have 28 audits in green 🤢. can you tell me if you made a change, so that I stop searching, lowered my LightHouse score again, I don't remember making any changes to my Website? 
[document.pdf](https://github.com/GoogleChrome/lighthouse/files/13928282/document.pdf)

**Comment by connorjclark** [comment]
You'll need to review our changelogs https://github.com/GoogleChrome/lighthouse/releases

If you have a more specific question, we can try to help.

**Comment by earthtopomaps** [comment]
Before in December 2023 the audits were in green in the 4 audits with a red arrow as in this photo attached here, now they are no longer in green. Yet the Lighthouse score is good! Do you see before they were in (PASSED AUDITS), now they are in (DIAGNOSTICS)?
![Test LightHouse screenshot](https://github.com/GoogleChrome/lighthouse/assets/22265545/1d34d0f0-be94-4446-a038-d814889ad627)
Thank You!

**Reply by adamraine** [reply, marked as answer]
This was an intentional change to performance diagnostic scoring. The audits are still passing, but are still displayed for informational purposes. Performance diagnostics do not directly affect the performance score.
